source/linux/Foundational_Components/U-Boot/UG-Network-K3.rst

```diff
@@ -50,48 +50,60 @@ binaries files to be sent over TFTP are listed in the table below.
 
-  .. code-block:: text
+   .. important::
 
-      subnet 10.0.0.0 netmask 255.0.0.0
-      {
-        range dynamic-bootp 10.0.0.2 10.0.0.16;
-        if substring (option vendor-class-identifier, 0, 16) = "TI K3 Bootp Boot"
-        {
-          filename "tiboot3.bin";
-        } elsif substring (option vendor-class-identifier, 0, 20) = "AM64X U-Boot R5 SPL"
-        {
-          filename "tispl.bin";
-        } elsif substring (option vendor-class-identifier, 0, 21) = "AM64X U-Boot A53 SPL"
-        {
-          filename "u-boot.img";
-        }
+      - Ethernet RGMII boot is supported over RGMII on J722s SoC.
+      - CPSW PHYs should be strapped as per ROM's expectation described in part's TRM.
+      - When the link info Bootmode pin is enabled, this means no auto-negotiation or reading of the Ethernet PHY is needed since the ROM will assume the link is up at 1Gbps, full duplex mode.
+      - Disable link info Bootmode pin so that ROM can identify the PHY and establishes link with the supported speed and duplex mode.
 
-        range 10.0.0.17 10.0.0.25;
-        default-lease-time 60000;
-        max-lease-time 720000;
-        next-server 10.0.0.1;
-      }
+.. ifconfig:: CONFIG_part_variant in ('AM62PX')
 
-.. ifconfig:: CONFIG_part_variant in ('AM62X')
+   Build tiboot3.bin using ``am62px_evm_r5_ethboot_defconfig``. Build tispl.bin and u-boot.img using ``am62px_evm_a53_ethboot_defconfig``. For instructions to build the bootloader images please refer to :ref:`Build-U-Boot-label`.
+
+   .. important::
+
+      - Ethernet RGMII boot is supported over RGMII on AM62px SoC.
+      - CPSW PHYs should be strapped as per ROM's expectation described in part's TRM.
+      - When the link info Bootmode pin is enabled, this means no auto-negotiation or reading of the Ethernet PHY is needed since the ROM will assume the link is up at 1Gbps, full duplex mode.
+      - Disable link info Bootmode pin so that ROM can identify the PHY and establishes link with the supported speed and duplex mode.
+
+.. ifconfig:: CONFIG_part_variant in ('J721S2')
+
+   Build tiboot3.bin using ``am68_evm_r5_ethboot_defconfig``. Build tispl.bin and u-boot.img using ``am68_evm_a72_ethboot_defconfig``. For instructions to build the bootloader images please refer to :ref:`Build-U-Boot-label`.
+
+   .. important::
+
+      - Ethernet RGMII boot is supported over RGMII on J721S2 SoC.
+
+.. ifconfig:: CONFIG_part_variant in ('J784S4')
+
+   Build tiboot3.bin using ``am69_evm_r5_ethboot_defconfig``. Build tispl.bin and u-boot.img using ``am69_evm_a72_ethboot_defconfig``. For instructions to build the bootloader images please refer to :ref:`Build-U-Boot-label`.
+
+   .. important::
+
+      - Ethernet RGMII boot is supported over RGMII on J784S4 SoC.
+
+.. ifconfig:: CONFIG_part_variant in ('AM64X','AM62X','J722S','AM62PX','J721S2','J784S4')
 
   If using ISC dhcpd an example host entry would look like this:
```

Comment:
Any plans to add [kea config examples](https://www.isc.org/kea/) since [ISC dhcpd has been deprecated for some time now](https://www.isc.org/dhcp/)?